### PR TITLE
E2e enhancements

### PR DESCRIPTION
**Pull request by kkoskelin**
This changeset has some good bones, but I think I need to find a better way to seed the s3 storage from the fixtures folder without having to hard-code the name of the local S3 bucket "noop-documents-local-us-east-1".  Looking for a second set of eyes.

### CODE REVIEW

**Review by dknesek — COMMENTED** [review]
One thing we need to do, Kris, is get a handle on when to do e2e test versus integration tests that go from presenter (cerebral) to the edge (but not including) the DB. One of the core ideas of CA is that we no longer include UIs and databases in our tests. The e2e tests should really be limited to smoke tests.  Where we are really lacking at the moment are tests on the use case interactors.

File: efcms-service/serverless.yml [hunk 1]
```diff
@@ -245,6 +247,72 @@ package:
 
 resources:
   Resources:
+    LocalDynamoDbTableDocuments:
```
Comment (sadlerw) [inline efcms-service/serverless.yml]: @codyseibert is this ok or should we put something in the package.json to create these tables? I'd like to keep the serverless yaml as small as possible if we can

**Review by sadlerw — APPROVED** [review]

File: efcms-service/serverless.yml (hunk 1 above)
Comment (codyseibert) [reply efcms-service/serverless.yml]: @kkoskelin we decided against creating the tables in serverless.  We actually already have a npm script called "npm run create-tables" which will create these same tables if dynamodb local is running.

File: efcms-service/package.json [hunk 2]
```diff
@@ -41,10 +41,11 @@
   "scripts": {
     "postinstall": "npm run install:dynamodb",
     "install:dynamodb": "./node_modules/.bin/sls dynamodb install -s local -r us-east-1 --domain noop --accountId noop --documentsTableName efcms-documents-local --casesTableName efcms-cases-local",
+    "_s3:seed": "cp -R storage/fixtures/s3/ storage/s3/noop-documents-local-us-east-1",
     "install:lambda": "docker pull lambci/lambda",
     "start:dynamodb": "./node_modules/.bin/sls dynamodb start -p 8000  --migrate true --stage local",
-    "start:local": "SLS_DEPLOYMENT_BUCKET=noop ./node_modules/.bin/sls offline start --noTimeout --stage local --region us-east-1 --domain noop  --documentsTableName efcms-documents-local --casesTableName efcms-cases-local --accountId noop",
-    "start": " $(sleep 5; npm run create-tables) & npm run start:local",
+    "start:local": "SLS_DEPLOYMENT_BUCKET=noop ./node_modules/.bin/sls offline start --noTimeout --stage local --region us-east-1 --domain noop --seed test --documentsTableName efcms-documents-local --casesTableName efcms-cases-local --accountId noop",
+    "start": " $(sleep 5; npm run _s3:seed > /dev/null) & npm run start:local",
```
Comment (codyseibert) [inline efcms-service/package.json]: we'd need to add back create-tables since we don't want serverless to know about dynamodb

File: efcms-service/serverless.yml (hunk 1 above)
Comment (kkoskelin) [reply efcms-service/serverless.yml]: @codyseibert yes, however it doesn't actually seed the database...? it creates empty tables.

**Comment by codyseibert** [comment]
🎈🎈🎈

**Review by codyseibert — APPROVED** [review]
